### PR TITLE
Left align the connect page contents

### LINKED ISSUE #2271: Left align the connect account page
https://github.com/Automattic/woocommerce-payments/pull/2196# has been merged with an old design (centered).

The text should be left-aligned as shown https://github.com/Automattic/woocommerce-payments/pull/1600

### PR DESCRIPTION
**Pull request by moon0326**
Fixes #2271 

#### Changes proposed in this Pull Request

This PR removes `text-align:center` style to left-align the contents on the page.

#### Testing instructions

1. Clone this repository and run `npm start`
2. Navigate to the Payments page and confirm that the page is left-aligned.

Screen Shot 2021-06-18 at 1 56 21 PM

### CODE REVIEW

**Comment by moon0326** [comment]
Putting it into a draft mode for now as Elizabeth pointed out that there might be a style change. I will compare my PR against the original and re-open it just in case.

**Comment by dechov** [comment]
Not sure if it's a big deal, depending on the precision of the design here, but it looks like the centering wasn't the only thing carried over from the old design to the new: the line height for the top header looks different between this version and https://github.com/Automattic/woocommerce-payments/pull/1600. (Looking at the change in https://github.com/Automattic/woocommerce-payments/pull/2196, that seems like it was the only other thing carried over, but I can't be certain.)

**Comment by moon0326** [comment]
> Not sure if it's a big deal, depending on the precision of the design here, but it looks like the centering wasn't the only thing carried over from the old design to the new: the line height for the top header looks different between this version and #1600. (Looking at the change in #2196, that seems like it was the only other thing carried over, but I can't be certain.)

Thank you for checking it out. No code has been changed as far as I know. I will double-check my "centered" PR against the original and see if there's a chance.

**Comment by moon0326** [comment]
@dechov @elizaan36 I downloaded the latest WC Payments plugin and assigned myself to the treatment group. I see the centered design from the latest WC Payments (screenshot below).

Screen Shot 2021-06-18 at 3 32 15 PM

Could you double-check, please? I'm not sure there was a change in the design at the last minute or not. @adrianduffell Could you also check, please?

**Comment by moon0326** [comment]
@dechov @elizaan36  In the meantime, I've updated the `line-height` css.

New screenshot:

Screen Shot 2021-06-18 at 3 43 17 PM

**Comment by dechov** [comment]
@moon0326 Looks good. Referencing https://github.com/Automattic/woocommerce-payments/issues/1436#issuecomment-824973580, there should be a comma in the ToS blurb – but also it should say "Finish setup" with a lowercase 's' (see https://github.com/Automattic/woocommerce-payments/issues/1323)

**Comment by moon0326** [comment]
> @moon0326 Looks good. Referencing [#1436 (comment)](https://github.com/Automattic/woocommerce-payments/issues/1436#issuecomment-824973580), there should be a comma in the ToS blurb – but also it should say "Finish setup" with a lowercase 's' (see #1323)

Thank you. I've updated comma in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/fe2bc49fd2e38aee777c327f1e8c8feb171df6a9 and lowercase 's' in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/a3e54f45e58dcd03292d6dabf5a8580c96170636

**Comment by elizaan36** [comment]
> Could you double-check, please? Was there a last-minute design change by any chance?

There wasn't a last minute design change, just something odd must have happened after #1600. Regardless, the text should be left-aligned as we roll this out to all users. [This screenshot](https://github.com/Automattic/woocommerce-payments/pull/2272#issuecomment-864306746) looks great.

File: client/connect-account-page/strings.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default {
 
 	terms: createInterpolateElement(
 		__(
-			'By clicking “Finish Setup” you agree to the <a>Terms of Service</a>',
+			'By clicking “Finish Setup”, you agree to the <a>Terms of Service</a>',
```
Comment (dechov) [inline client/connect-account-page/strings.js:64]: Could the "Finish setup" lowercase be reflected here too?

**Review by dechov — APPROVED** [review]
Looks good – left one minor comment, and also need to update one snapshot (`npm run test:update-snapshots`), and then good to merge!

File: client/connect-account-page/strings.js (hunk 1 above)
Comment (moon0326) [reply client/connect-account-page/strings.js]: Updated in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/ad664d1738ff13a20b0a8c853d34626537736eb1. Snapshop is also updated in https://github.com/Automattic/woocommerce-payments/pull/2272/commits/533cd0f4d09a3022a97d8db6ec9103e1f4d2c210

File: client/connect-account-page/test/index.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ describe( 'ConnectAccountPage', () => {
 		};
 
 		render( <ConnectAccountPage /> );
-		user.click( screen.getByRole( 'link', { name: /Finish Setup/ } ) );
+		user.click( screen.getByRole( 'link', { name: /Finish setup/ } ) );
```
Comment (dechov) [inline client/connect-account-page/test/index.js:49]: Just noticed this too – 👍!

**Pull request merged**